### DISCUSSION TITLE
Using Google font icons

### DISCUSSION THREAD

**Discussion by kla-ko** [GeorgianStan/vanilla-context-menu#27] [Q&A]
Hi
I want to use google icon fonts to add icons to menu items. The samples in this repo are showing how to use Font Awesome, which results in html like ``<i class="fa fa-face"></i>``.

It seems that google font icons have to be translated differently, like ``<i class="material-icons">face</i> ``.

Is it possible to achieve this, or is there another way to use google icons?

Thanks

**Comment by GeorgianStan** [comment]
Hi @kla-ko! You are right, I didn't notice that. I will update the library and eventually will add the option to supply the icon as a string. Im this way a valid HTML tag can be used and the library will run sanitization internally for the supplied string.

**Reply by GeorgianStan** [reply]
I updated the library. The new version 1.3.0 includes a new field named `iconHTML` that you  can use for this. I also updated the documentation and the demo to include a Material Icon.

**Reply by kla-ko** [reply]
This is great! Thanks. I will try asap ...
